src/node_crypto.cc

```diff
@@ -255,13 +255,53 @@ void ThrowCryptoError(Environment* env,
                       unsigned long err,  // NOLINT(runtime/int)
                       const char* default_message = nullptr) {
   HandleScope scope(env->isolate());
```

Comment:
Also. Do i need to change any documentation on this? if so, where is it located?